lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -389,7 +389,7 @@ def test_delete_student_attempts(self):
                 course_id=self.course_key,
                 module_state_key=msk
             ).count(), 1)
-        reset_student_attempts(self.course_key, self.user, msk, delete_module=True)
+        reset_student_attempts(self.course_key, self.user, msk, delete_module=True, requesting_user=self.user)
```

Comment:
Nit: I'd prefer params are ordered as they are declared if there is no reason not to, but it is fine as is.